setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = pyflow
+version = attr: pyflow/version.__version__
+author = European Centre for Medium-Range Weather Forecasts (ECMWF)
+author_email = software.support@ecmwf.int
+license = Apache 2.0
+license_files = LICENSE
+description = Create pythonic ecFlow suites
+long_description = file: README.md
+long_description_content_type=text/markdown
+url = https://pyflow-workflow-generator.readthedocs.io/en/
+
```

Comment:
Missing platform, classifiers, and other project information that appears in the PyPI project page. Maybe another improvement for this change.